### PR TITLE
Add an example of how to configure prometheus and grafana for stardog metrics

### PR DESCRIPTION
**Pull request by igable**

### CODE REVIEW

File: examples/grafana/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+Stardog Grafana Dashboard
+=========================
+
+Stardog provides a preconfigured Stardog dashboard for Grafana in the form of a json files which can
+be imported into Grafana. In order to be able to use the dashboard you must have Prometheus
+configured to scrape Stardog kubenetes pods for metrics data.
```
Comment (pdmars) [inline examples/grafana/README.md]: s/kubenetes/kubernetes (k8s)

File: examples/grafana/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+------------------------------------------------
+
+We recommend that you create a Stardog user specifically for monitoring. This user will not be
+able to read data but will be able to access the Prometheus monitoring endpoint:
+
+    stardog-admin user add monitor
+
+If you don't have Prometheus deployed already on your K8S cluster we recommend deploying using
+[Prometheus Helm Hub](https://hub.helm.sh/charts/stable/prometheus) templates. To scape Stardog pods
+with Prometheus you must create addition scrape jobs. When using Prometheus deployed with Helm the
```
Comment (pdmars) [inline examples/grafana/README.md]: additional

File: examples/grafana/README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+simplest way is to pass in additional scaraping configs using a helm values file
```
Comment (pdmars) [inline examples/grafana/README.md]: s/scaraping/scraping

File: examples/grafana/README.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+- [stardog-overview.json](https://raw.githubusercontent.com/stardog-union/stardog-examples/develop/examples/grafana/stardog-overview.json)
+- [stardog-databases.json](https://raw.githubusercontent.com/stardog-union/stardog-examples/develop/examples/grafana/stardog-databases.json)
+
+Download via curl:
+
+    curl -o stardog-overview.json https://raw.githubusercontent.com/stardog-union/stardog-examples/develop/examples/grafana/stardog-overview.json
+    curl -o stardog-databases.json https://raw.githubusercontent.com/stardog-union/stardog-examples/develop/examples/grafana/stardog-databases.json
+
+The follow the [Grafana instructions](https://grafana.com/docs/grafana/latest/reference/export_import/#importing-a-dashboard)
```
Comment (pdmars) [inline examples/grafana/README.md]: s/The/Then

File: examples/grafana/README.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+be imported into Grafana. In order to be able to use the dashboard you must have Prometheus
+configured to scrape Stardog kubenetes pods for metrics data.
+
+![Stardog Overview](img/Stardog_Overview_-_Grafana_1.png?raw=true "Stardog Overview")
+
+Step 1: Configuring Prometheus to scrape Stardog
+------------------------------------------------
+
+We recommend that you create a Stardog user specifically for monitoring. This user will not be
+able to read data but will be able to access the Prometheus monitoring endpoint:
```
Comment (pdmars) [inline examples/grafana/README.md]: suggestion, maybe reword: This user will be able to access the Prometheus endpoint.

**Review by pdmars — CHANGES_REQUESTED** [review]
Looks good. Few reword suggestions and spelling mistakes.

File: examples/grafana/README.md (hunk 1 above)
Comment (igable) [reply examples/grafana/README.md]: fixed

File: examples/grafana/README.md (hunk 2 above)
Comment (igable) [reply examples/grafana/README.md]: fixed

File: examples/grafana/README.md (hunk 4 above)
Comment (igable) [reply examples/grafana/README.md]: fixed

File: examples/grafana/README.md (hunk 5 above)
Comment (igable) [reply examples/grafana/README.md]: done

**Review by pdmars — APPROVED** [review]
Looks great, thanks guys!

**Review by buzztroll — APPROVED** [review]
